### ISSUE TITLE
Variable Smart Rewind

### ISSUE THREAD

**Issue by Lego296** [TortugaPower/BookPlayer#976]
## Feature request

I really enjoy the smart rewind option, but please make it more customizable. 

30 seconds is a lot, switch between 10/20/30 sec should work for most users

**Comment by gmaddry** [comment]
I agree this would be useful. Even better for dynamic option would allow setting the time for rewind.

**Comment by BartemX** [comment]
Maybe also add an option to always rewind 2 seconds after a pause, Voice does this on android and it's one feature I miss.

**Comment by GianniCarlo** [comment]
this is now addressed, you can set the interval, and also the amount we rewind per time passed is different, so it should feel more natural, thanks again @lyallcooper !
